### ISSUE TITLE
Unity VR Samples

### ISSUE THREAD

**Issue by steven-king** [steven-king/mj588#15]
https://blogs.unity3d.com/2015/12/09/get-started-with-vr-sample-pack-learning-articles/

Work through this and understand how each game/scene works.

**Comment by steven-king** [comment]
UPDATE: Please go through the part that is labeled: Introduction to VR (deprecated) Parts 1-5. It is not actually deprecated, they just updated it with the videos and new assets. But the problem is the new assets are NOT free. 
SO:
Download these samples to play with and try to understand:
https://assetstore.unity.com/packages/essentials/tutorial-projects/vr-samples-51519

Then walk through these tutorials that I have highlights in yellow found here https://unity3d.com/learn/tutorials/s/virtual-reality.
<a href="https://ibb.co/eUD6Lx"><img src="https://preview.ibb.co/mJA8YH/Screen_Shot_2018_02_23_at_8_06_43_AM.png" alt="Screen_Shot_2018_02_23_at_8_06_43_AM" border="0"></a>

**Comment by sivonne** [comment]
For tomorrow's class, is this the main thing to have done? As well as playing with lighting in our scenes?

**Comment by steven-king** [comment]
Yes. That is correct. Also, don’t forget about keeping up with the Drone training.

On Feb 27, 2018, at 9:36 PM, sivonne <notifications@github.com<mailto:notifications@github.com>> wrote:


For tomorrow's class, is this the main thing to have done? As well as playing with lighting in our scenes?
